RusimbiPatrick/Banka
# #165310858  built create bank account endpoint

**What does this PR do?**

It helps a user to create a bank account.

**Description of Task to be done?**

It adds a controller for creating a bank account.
It adds a route for creating a bank account.
It adds a service for creating a bank account
It adds a User input validation on creating a new account(in the middleware forder )

**How should this be manually tested?**

Start the Server and POSTMAN.
Browse the following endpoint route: localhost:3000/api/v1/accounts.
Submit the required information with POST method.
Check if you receive a message telling you that New ${type} account created successfully with a 201 HTTP Status Code.
